src/main.js

```diff
@@ -200,34 +196,28 @@ function showMyPoster() {
   posterQuote.innerText = currentPoster.quote;
   posterImage.src = currentPoster.imageURL;
 }
-  console.log(currentPoster);
-
-// var newImage = new Poster(imageInput.value, titleInput.value, quoteInput.value)
+  // console.log(currentPoster);
 
-//create function that creates new object instance of Poster class
-//with the last element on each array
+// iteration three:
 
+function addGrid() {
+  var posterHTML = '';
 
+  for (var i = 0; i<savedPosters.length; i++) {
+    posterHTML += `<article class="mini-poster">
```

Comment:
As discussed we will continue to review this syntax for other possible ways to refactor.